app/components/find/courses/training_locations/view.html.erb

```diff
@@ -0,0 +1,25 @@
+    <dl class="govuk-summary-list">
```

Comment:
Can we use the govuk component instead of just using the classes?
https://govuk-components.netlify.app/components/summary-list/#summary-lists-without-actions

Reply:
Great suggestion thanks, I was looking for this.

Have updated here: https://github.com/DFE-Digital/publish-teacher-training/pull/4522/commits/3baa34cb182800df71fcd6dcfe9eb2e55a0e2566